ios/MarigoldSDKReactNativeTests/RNMessageStreamTests.m

```diff
@@ -233,6 +235,43 @@ -(void)clearMessages:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectB
             [[check should] equal:error];
         });
     });
+    context(@"the useDefaultInAppNotification method", ^{
+        it(@"should set the default value as YES", ^{
+            [rnMessageStream useDefaultInAppNotification:YES];
+            
+            [[theValue(rnMessageStream.defaultInAppNotification) should] equal:theValue(YES)];
+        });
+        it(@"should set the default value as NO", ^{
+            [rnMessageStream useDefaultInAppNotification:NO];
+            
+            [[theValue(rnMessageStream.defaultInAppNotification) should] equal:theValue(NO)];
+        });
+    });
+    context(@"the shouldPresentInAppNotificationForMessage method", ^{
```

Comment:
I think this test needs to be reworked. The default return value is `NO` anyway, which I think is what is being returned here. The notify call, the wait and the check should be moved out of the dispatch, `shouldPresentInAppNotificationForMessage` will block until the timeout or the `notify` is received. Worth checking that notifying `NO` returns `YES`, and adding a separate test for the timeout case.